From 66a6082634687e45b321a5ade9887420d5830162 Mon Sep 17 00:00:00 2001
From: Lubos Kardos <lkardos@redhat.com>
Date: Fri, 15 Apr 2016 10:47:57 +0200
Subject: [PATCH 1/2] Add %{_default_patch_flags} to %__patch which is used in
 %autosetup

%{_default_patch_flags} is used in %patch spec directive and now it is
used also in %__patch macro which is used in %autosetup macro. This make
consitent using newer %autosetup macro and older %patch directives.
---
 macros.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/macros.in b/macros.in
index ce252d4..bbff0c1 100644
--- a/macros.in
+++ b/macros.in
@@ -1085,7 +1085,7 @@ done \
 # Plain patch (-m is unused)
 %__scm_setup_patch(q) %{nil}
 %__scm_apply_patch(qp:m:)\
-%{__patch} %{-p:-p%{-p*}} %{-q:-s} --fuzz=%{_default_patch_fuzz}
+%{__patch} %{-p:-p%{-p*}} %{-q:-s} --fuzz=%{_default_patch_fuzz} %{_default_patch_flags}
 
 # Mercurial (aka hg)
 %__scm_setup_hg(q)\
-- 
1.9.3

